WordPress/UITests/Tests/DashboardTests.swift

```diff
@@ -28,4 +28,15 @@ class DashboardTests: XCTestCase {
             .goToPlanSelection()
             .verifyPlanSelectionScreenLoaded()
     }
+
+    func testPagesCardHeaderNavigation() throws {
+        try MySiteScreen()
+            .scrollToPagesCard()
+            .verifyPagesCard()
+            .verifyPagesCard(hasPage: "Blog")
+            .verifyPagesCard(hasPage: "Shop")
+            .verifyPagesCard(hasPage: "Cart")
```

Comment:
i wonder if we need to have three pages cards for this test. would be fine to test with just 2? (to test single vs multiple cases, that way it's less time for test validation and fewer files to maintain)